src/entity_gui.cpp

```diff
@@ -17,17 +17,15 @@ static const WidgetPart _entity_remove_widgets[] = {
 			Widget(WT_CLOSEBOX, INVALID_WIDGET_INDEX, COL_RANGE_RED),
 		EndContainer(),
 		Widget(WT_PANEL, INVALID_WIDGET_INDEX, COL_RANGE_RED),
-			Intermediate(2,0),
+			Intermediate(2, 1),
 				Widget(WT_CENTERED_TEXT, ERW_MESSAGE, COL_RANGE_RED),
 						SetData(GUI_ENTITY_REMOVE_MESSAGE, STR_NULL), SetPadding(5, 5, 5, 5),
-				EndContainer(),
 			Intermediate(1, 5), SetPadding(0, 0, 3, 0),
```

Comment:
Much cleaner already :)

Since this is the start of the 2nd row, it belongs under the start of the first row, at line 21